src/cljc/matcher_combinators/matchers.cljc

```diff
@@ -1,6 +1,11 @@
 (ns matcher-combinators.matchers
   (:require [matcher-combinators.core :as core]))
 
+(defn value
```

Comment:
is there a better name? perhaps `reference` or `exact`?
Will this cause confusion because it is quite similar to `equals`?
Perhaps it is better to introduce map/seq specific value matcher like `map-reference` or something